tests/kafkatest/services/kafka/kafka.py

```diff
@@ -407,6 +411,7 @@ def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAI
             kraft_broker_configs = {
                 config_property.PORT: config_property.FIRST_BROKER_PORT,
                 config_property.NODE_ID: self.idx(node),
+                config_property.NEW_GROUP_COORDINATOR_ENABLE: use_new_coordinator
             }
```

Comment:
dont we want to conditionally add this when `use_new_coordinator` is true?
```suggestion
            kraft_broker_configs = {
                config_property.PORT: config_property.FIRST_BROKER_PORT,
                config_property.NODE_ID: self.idx(node)
            }
            if self.use_new_coordinator:
                kraft_broker_configs[config_property.NEW_GROUP_COORDINATOR_ENABLE] = use_new_coordinator
            }
```
that way we stick with the default value?  Or do we care?  Either was is fine.

Reply:
I figured the value get's overwritten anyways on start and here it's gonna be false at init by default anyways so just assigned it until the overwrite happens if its set to true